src/windows/windows.js

```diff
@@ -10,7 +11,7 @@ exports.createTimerWindow = () => {
   }
 
   let {width, height} = electron.screen.getPrimaryDisplay().workAreaSize
-  timerWindow = new electron.BrowserWindow({
+  timerWindow = createBrowserWindowOnCurrentScreen({

```

Comment:
It doesn't seem like calling the helper method here does anything significant. I can't seem to get the small timer window to be above full-screen apps. And if it did launch in a full screen space then it would follow that app around which seems a bit odd to me.

Reply:
I don't have access to my mac but when I do I will look into this. Since it was a while since I did the changes I have a hard time remembering which tests I performed.